src/ui/lib/components/button-dense.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import { Button, type ButtonProps } from '@ui';
+	import { current_component } from 'svelte/internal';
+
+	export let dense = true;
+
+	interface $$Props extends ButtonProps {
+		dense?: boolean;
+	}
+
+	$: computedClass = `${(dense && 'dense') || ''} ${$$restProps.class || ''}`;
+</script>
+
+<Button eventComponent={current_component} {...$$restProps} class={computedClass}>
+	<slot />
+</Button>
+
+<style global>
+	.box.button.dense {
+		height: 30px !important;
+		padding: 4px 10px !important;
+		font-size: small !important;
+	}
+</style>
```

Comment:
Is there an issue / PR on unicorn UI for this custom component, doesn't look like it would be a huge lift for @TristanHoladay to implement.

Reply:
https://github.com/defenseunicorns/UnicornUI/issues/197